src/gpo/assets/en-US/PowerToys.adml

```diff
@@ -244,6 +251,8 @@ If you disable or don't configure this policy, no predefined rules are applied.
       <string id="MwbValidateRemoteIp">Validate remote machine IP Address</string>
       <string id="MwbDisableUserDefinedIpMappingRules">Disable user defined IP Address mapping rules</string>
       <string id="MwbPolicyDefinedIpMappingRules">Predefined IP Address mapping rules</string>
+      <string id="AllowPowerToysAdvancedPasteOnlineAIModels">Advanced Paste: Allow using online AI models</string>
+      <string id="AllowDiagnosticData">Allow sending diagnostic data</string>
```

Comment:
According to the description it makes more sense that it is named "Disable ...". The policy allows only disabling. Maybe it makes sense to change the behavior to "Telemetry disabled" if the policy is enabled.

Reply:
aligned with how OS treat the same thing